### PR TITLE
navbar 비율 조정 및 Sign In 없어지는 이슈 수정, 전체적인 비율 수정

### PR DESCRIPTION
**Pull request by poohtime1**
요청하신 이슈 수정 및 브라우저 사이즈를 줄였을때 생기는 비율 버그를 수정하였습니다.

### CODE REVIEW

File: frontend/src/App.js
Comment (avelynhc) [inline frontend/src/App.js:1]: Is there any reason why we do not use a CSS file for style? If not, can we make it consistent and extract every style-related code to a CSS file?"

File: frontend/src/components/GamesCards.jsx
Comment (avelynhc) [inline frontend/src/components/GamesCards.jsx:1]: Is there any reason why we do not use a CSS file for styles? If not, can we make it consistent and extract every style-related code to a CSS file?"

File: frontend/src/components/Navbar.jsx
Comment (avelynhc) [inline frontend/src/components/Navbar.jsx:1]: Please remove space on line 18.

File: frontend/src/components/Navbar2.jsx
Comment (avelynhc) [inline frontend/src/components/Navbar2.jsx:1]: Is there any reason why we do not use a CSS file for styles? If not, can we make it consistent and extract every style-related code to a CSS file?"

File: frontend/src/components/RecentlyCards.jsx [hunk 1]
```diff
@@ -4,49 +4,20 @@ const RecentlyCards = () => {
...
-            </div>
-
-            {/*Card */}
-            <div className='rounded-xl relative'>
-
-               {/*Overlay*/}
-               <div className='absolute w-full h-full bg-black/50 rounded-xl'>
-
-                  <button className='border-white bg-amber-400 text-white mx-2 absolute bottom-4'>GAME START</button>
+                  <button style={{"width": "135px"}} className='border-white bg-amber-400 text-white mx-2 absolute bottom-4'>GAME START</button>
```
Comment (avelynhc) [inline frontend/src/components/RecentlyCards.jsx:12]: Is there any reason why we do not use a CSS file for styles? If not, can we make it consistent and extract every style-related code to a CSS file?

**Review by avelynhc — CHANGES_REQUESTED** [review]
Could you please add @HyoanMoon for frontend code review going forward? Thanks!

**Review by HyoanMoon — COMMENTED** [review]
Can you resolve conflict?

**Comment by avelynhc** [comment]
@poohtime1  @HyoanMoon 
These are a few things to consider if you do NOT want to use a CSS file:
- In-line styles are a bit faster but performance difference is very low.
- How are you going to implement reusable style code? A bit redundant.
- We already have the Index.css file, so the CSS file needs to be downloaded anyway.
- The browser will cache the CSS file for the first time, so it won't affect performance later."

**Pull request merged**